fluidefi/fluidefi-caspernet-aggregator-tools
# Block Hours Consumer

**Summary of change:**
- Added a new micro-service, block hours consumer, which uses all of the block hour micro-services in one class. 

**List of tasks:** 


**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/block-hours-consumer`
```bash
git checkout feature/block-hours-consumer
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm block-hour-consumer test has passed.
- Run example.
